feature-configs/base/sriov/04-sriov-networknodepolicy.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriov-network-node-policy
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice
+  isRdma: false
+  nicSelector:
+    pfNames:
+      - eno1
```

Comment:
Now that the policy is under base, you can provide a kustomization example under demo to change something (numVfs / pfName for example)

Reply:
> provide a kustomization example under demo to change something

not sure if this repo is for showing kustomize features... IMHO we should just provide good defaults (in case there are such) in base, and then specific values which are really targeting our CI and demo environments #justMy2Cents

Reply:
Thanks @slintes. I'll try to explain better: I did not mean to show kustomize features, but an example of how a real user would use kustomize for deploying the same network definition on different environments, maintaining the same structure.
That's why I thought the pfName was a good example (and it will be probably different between envs). Same for vf#, it may depend on the nic shipped on the environment.